xkcd.py

```diff
@@ -42,6 +42,13 @@
     Image = None
 
 
+@dataclass
+class MSC2530MediaMessageEventContent(MediaMessageEventContent):
+    filename: str = None
```

Comment:
This should just use the fields in mautrix-python and bump up the minimum maubot version to 0.4.3 in maubot.yaml